src/app/data-explorer/query-builder-wrapper/query-builder-wrapper.component.html

```diff
@@ -26,50 +26,8 @@
       [config]="config"
       [operatorMap]="operatorMap"
       [allowRuleset]="allowRuleset"
+      [isDataQuery]="isDataQuery()"
```

Comment:
When binding a method to a property in Angular, the method gets run upon every change detection cycle (possibly every keypress/mouse move/http event/etc) because Angular has to run the method in order to determine if it's value has changed. This is a relatively inexpensive method call, but it will still be run many more times than necessary. Instead best practice is to use a class property like `isDataQuery` because then Angular can do a simple 'has the value changed' check on it's own rather then run the method and then perform that check.

Reply:
Changed to be a boolean variable